istioctl/OWNERS

```diff
@@ -0,0 +1,62 @@
+reviewers:
```

Comment:
This is copied from pilot/OWNERS and we can probably trim it down a bit.

Reply:
We should. Also, ideally it should be in alphabetical order (I know you copied this, but lets try to enforce for new files too).